src/applications/personalization/dashboard/reducers/claimsV2.js

```diff
@@ -1,27 +1,29 @@
 import merge from 'lodash/merge';
 
-import set from 'platform/utilities/data/set';
+import set from '@department-of-veterans-affairs/platform-forms-system/set';
```

Comment:
Random observation, I didn't know this `set` utility existed... oof.

This existing kind of stinks because this set function has a different order for it's arguments than the lodash's set function, which is what I have been using all along, so that is bound to cause some confusion (at least for me).

 The forms-system one is `set(path, value, object)` vs lodash does `set(object, path, value)`
 
 That being said, the forms-system version of set is used like 120+ times throughout the code base and the lodash one is only used a handful of time. TIL

Reply:
thanks for bringing this up!! i didn't even notice that. using lodash's set function instead, since we're not dealing any forms here

Reply:
documented in this future ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/75917